### ISSUE TITLE
Axon recipe is selecting the wrong driver.

### ISSUE THREAD

**Issue by fdutton** [Tripwire/chef-tripwire_agent#37]
The latest TGZ from Tripwire contains drivers for DKMS, RHEL and SUSE. The code on line 151 of axon.rb appears to select the last matching entry in the directory (in lexicographical order), which is SUSE. The result is that this Chef recipe attempts to install the SUSE driver on every machine. I tried it on 30 RHEL machines and it failed on every machine.

```ruby
driver_basename = files.find { |item| item.include?(new_resource.use_dkms_driver ? 'driver-dkms' : 'driver') }
```
I'm new to both Chef and Ruby so forgive me if this is something I should be able to discern for myself.

**Comment by pfaffle** [comment]
Sorry for the delayed response. That does certainly look like a bug to me. I'll see if I or someone else can find some time to work on it in the near future.

**Comment by bespokebob** [comment]
This was fixed in #39
